packages/deployments/contracts/contracts/core/connext/facets/BridgeFacet.sol

```diff
@@ -184,14 +145,6 @@ contract BridgeFacet is BaseConnextFacet {
     return s.relayerFees[_transferId];
   }
 
-  function routedTransfers(bytes32 _transferId) public view returns (address[] memory) {
-    return s.routedTransfers[_transferId];
-  }
```

Comment:
So really this is set on `execute` and used on `reconcile` -- I'm wondering if this belongs in the bridge or nomad facet. Currently leaning towards bridge because it's relevant only for transfers that have already been executed.

Reply:
I can put it back. In fact, having all the getters be on one contract may be a decent argument in of itself